tests/lightning_qubit/test_jacobian_method.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import PHI, THETA, LightningDevice, device_name  # tested device
+from pennylane.devices import DefaultExecutionConfig, DefaultQubit, ExecutionConfig
+from pennylane.tape import QuantumScript
+
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+    from pennylane_lightning.lightning_qubit.lightning_qubit import (
+        jacobian,
+        simulate,
+        simulate_and_jacobian,
+        simulate_and_vjp,
+        vjp,
+    )
+
+
+if device_name == "lightning.kokkos":
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
+    from pennylane_lightning.lightning_kokkos.lightning_kokkos import (
+        jacobian,
+        simulate,
+        simulate_and_jacobian,
+        simulate_and_vjp,
+        vjp,
+    )
+
+
+if device_name not in ("lightning.qubit", "lightning.kokkos"):
+    pytest.skip(
+        "Exclusive tests for lightning.qubit and lightning.kokkos. Skipping.",
+        allow_module_level=True,
+    )
+
+if not LightningDevice._CPP_BINARY_AVAILABLE:
+    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
+
+
+# General LightningStateVector fixture, for any number of wires.
+@pytest.fixture(
+    scope="module",
+    params=[np.complex64, np.complex128],
+)
+def lightning_sv(request):
+    def _statevector(num_wires):
+        return LightningStateVector(num_wires=num_wires, dtype=request.param)
+
+    return _statevector
+
+
+@pytest.mark.skipif(
+    device_name == "lightning.tensor",
+    reason="lightning.tensor does not support derivatives",
+)
```

Comment:
OK, for the future. But testing `lightning.tensor` would already be skipped in line 47.